data/kubevirt-ipam-controller/003-passtbindingcni.yaml

```diff
@@ -55,6 +55,6 @@ spec:
         - name: cnibin
           hostPath:
             path: {{ .CNIBinDir }}
-      nodeSelector: {{ toYaml .Placement.NodeSelector | nindent 8 }}
-      affinity: {{ toYaml .Placement.Affinity | nindent 8 }}
-      tolerations: {{ toYaml .Placement.Tolerations | nindent 8 }}
+      nodeSelector: {{ toYaml .PlacementPasst.NodeSelector | nindent 8 }}
```

Comment:
you also need to change it on the [bump script](https://github.com/kubevirt/cluster-network-addons-operator/blob/f1504bd54fc83f371010d8fbd858d51fea4686cc/hack/components/bump-kubevirt-ipam-controller.sh#L78-L80)